modules/architecture-platform-benefits.adoc

```diff
@@ -40,19 +32,16 @@ unique features and benefits of {product-title}.
 [id="architecture-custom-os_{context}"]
 == Custom operating system
 
-{product-title} uses {op-system-first}, a new container-oriented operating
+{product-title} uses {op-system-first}, a container-oriented operating
 system that combines some of the best features and functions of the CoreOS and
 Red Hat Atomic Host operating systems. {op-system} is specifically designed for
 running containerized applications from {product-title} and works with new tools
 to provide fast installation, Operator-based management, and simplified upgrades.
 
 {op-system} includes:
 
-* Ignition, which is a firstboot system configuration for initially bringing up and
-configuring {product-title} nodes.
-* cri-o, a Kubernetes native container runtime implementation that integrates
-closely with the operating system to deliver an efficient and optimized
-Kubernetes experience.
+* Ignition, which {product-title} uses as a firstboot system configuration for initially bringing up and configuring machines.
+* CRI-O, a Kubernetes native container runtime implementation that integrates closely with the operating system to deliver an efficient and optimized Kubernetes experience. CRI-O provides facilities for running, stopping, and restarting containers. It fully replaces the Docker Container Engine in {product-title} {product-version}.
```

Comment:
This last sentence seems unclear. It's hard to tell whether you're saying that Docker Container Engine is in 4.2, or that in 4.2 it was replaced. Suggest rewording for clarity, maybe saying that it replaces Docker Container Engine, which was used in previous (maybe specifically say 3?) versions of OCP?